be/src/exec/vectorized/hdfs_scanner.h

```diff
@@ -53,6 +53,43 @@ struct HdfsScanStats {
     }
 };
 
+class CountedSeekableInputStream : public io::SeekableInputStreamWrapper {
```

Comment:
Why to put inner class into header file? You could put this clas and `open_random_access_file` into cpp file.

Reply:
done